### DISCUSSION TITLE
Hologram

### DISCUSSION THREAD

**Discussion by Steev93** [Zrips/CMI#3075] [Feature Requests]
Allow a confirm delete message to display so accidents do not happen! With /hologram

http://prntscr.com/rvgg3r

Is this possible?

**Comment by mrfloris** [comment]
While my argument could be: It's an admin tool, backups are always recommended of course, and since you're editing server stuff it's nice to pay extra close attention.

That said: We are all human and make mistakes, and boy if all the [x] delete things that can't be undone as a policy could have a confirmation option. Just like /cmi clear, to clear inventory with confirmation, I wouldn't mind to have /mail and /warps and /kits and commands in /aliaseditor etc all to have a confirmation to click on or type, just to help prevent issues.

Server owners that are super confident and do not wish for this, could perhaps use a global settings or permission like
```
- cmi.confirmation.player
- cmi.confirmation.staff
```
Or whatever, just to toggle on/off confirmation prompts on [x] stuff when it's either a player or helper/staff/op etc.

i vote yes.
